client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -13,7 +13,7 @@ import EditorTabs from './editor-tabs';
 const { showUpcomingChanges } = envData;
 
 const mapStateToProps = createStructuredSelector({
-  currentTab: currentTabSelector
+  currentTab: currentTabSelector as (state: unknown) => number
```

Comment:
@ojeytonwilliams This looks absolutely fine. However, I wonder if the reason it is needed is because of the `createStructuredSelector` function being different to what we normally use (`createSelector`)

Reply:
Thanks, Shaun.  Hopefully it'll magically fix itself once we migrate enough files.